scala/lambdas/rotate-preservation-system-password/src/test/scala/uk/gov/nationalarchives/rotatepreservationsystempassword/LambdaTest.scala

```diff
@@ -56,34 +56,34 @@ class LambdaTest extends AnyFlatSpec with EitherValues {
 
   "handler createSecret" should "not call putSecretValue if a Pending value already exists" in {
     val rotationEvent = RotationEvent(CreateSecret, "id", "token")
-    val secret = Secret(Map("token" -> List(SecretStage(Map("username" -> "password"), Pending))))
+    val secret = Secret(Map("token" -> List(SecretStage(Option(AuthDetails("username", "password", "")), Pending))))
 
     val (secretResult, _, _) = runLambda(rotationEvent, secret, Credentials("", ""))
 
     secretResult.versionToStage.size should equal(1)
-    secretResult.versionToStage("token").head.value should equal(Map("username" -> "password"))
+    secretResult.versionToStage("token").head.value should equal(Option(AuthDetails("username", "password", "")))
 
   }
 
   "handler createSecret" should "create the secret value if a Pending value does not already exist" in {
     val rotationEvent = RotationEvent(CreateSecret, "id", "token")
     val versionToStage = Map(
-      "token" -> List(SecretStage(Map.empty, Pending)),
-      "anotherToken" -> List(SecretStage(Map("user" -> "currentSecret"), Current))
+      "token" -> List(SecretStage(None, Pending)),
+      "anotherToken" -> List(SecretStage(Option(AuthDetails("user", "currentSecret", "")), Current))
     )
     val secret = Secret(versionToStage)
 
-    val (secretResult, _, _) = runLambda(rotationEvent, secret, Credentials("", ""))
+    val (secretResult, _, res) = runLambda(rotationEvent, secret, Credentials("", ""))
```

Comment:
You changed this from a `_` to `res` but it doesn't seem to be used; is that correct?

Reply:
I was debugging, I've put it back to _, cheers